cypress/support/fragments/inventory/inventorySearchAndFilter.js

```diff
@@ -99,13 +100,31 @@ const searchHoldingsByHRID = (hrid) => {
   InventoryInstances.waitLoading();
 };
 
+const searchInstanceByKeyword = (keyword) => {
+  cy.do([
+    Select({ id: 'input-inventory-search-qindex' }).choose(
+      'Keyword (title, contributor, identifier, HRID, UUID)',
+    ),
+    TextArea({ id: 'input-inventory-search' }).fillIn(keyword),
+    searchButton.click(),
+  ]);
+};
+
 const searchInstanceByTitle = (title) => {
   cy.do([TextField({ id: 'input-inventory-search' }).fillIn(title), searchButton.click()]);
   InventoryInstance.waitLoading();
 
   return InventoryInstance;
 };
 
+const searchInstanceByOCLC = (oclc) => {
+  cy.do([
+    Select({ id: 'input-inventory-search-qindex' }).choose('OCLC number, normalized'),
+    TextArea({ id: 'input-inventory-search' }).fillIn(oclc),
+    searchButton.click(),
+  ]);
+};
+
```

Comment:
You can use method "InventoryInstances.searchInstancesWithOption()" instead the one you have created here. Please try to use already existing methods, try to create new method if there is no related method available.

Reply:
Fixed